medical/data/ir_sequence_data.xml

```diff
@@ -4,7 +4,7 @@
 
 <odoo>
     <data noupdate="1">
-        
+
```

Comment:
Please don't make non-functional, stylistic changes to code. This dirties the diff and unless there is a guideline specifically saying not to do something in terms of style, we should not change it. Each dev has their differing opinions on what may or may not look good, which creates somewhat of a style war at the cost of the size or our repo.